tests/models/test_functionconfig.py

```diff
@@ -31,8 +31,15 @@ def test_functionconfig() -> None:
         "functionType": "foo/bar",
     }
 
+    # NOTE: model_dump_json() returns a JSON string with no whitespace.
```

Comment:
```
# NOTE: The method 'model_dump_json()' returns a compact JSON string (no extra spaces or newlines).
#       To match its format, we use 'json.dumps()' with specific parameters to eliminate whitespace.

```

Reply:
The change I've done here has also been replicated for `test_secretconfig.py` and `test_triplestoreconfig.py`.